src/azure-cli/azure/cli/command_modules/acs/custom.py

```diff
@@ -2583,14 +2589,25 @@ def aks_get_versions(cmd, client, location):
 def aks_get_credentials(cmd, client, resource_group_name, name, admin=False,
                         path=os.path.join(os.path.expanduser(
                             '~'), '.kube', 'config'),
-                        overwrite_existing=False, context_name=None):
+                        overwrite_existing=False, context_name=None, public_fqdn=False):
     credentialResults = None
+    serverType = None
+    if public_fqdn:
+        serverType = 'public'
     if admin:
-        credentialResults = client.list_cluster_admin_credentials(
-            resource_group_name, name)
+        if serverType is None:
+            credentialResults = client.list_cluster_admin_credentials(
```

Comment:
what is the result if serverType == None?
            credentialResults = client.list_cluster_admin_credentials(
                resource_group_name, name, serverType)

I am asking to check if we can remove this if/else.

Reply:
Add this if/else because azure stack has integration test with old api version. Otherwise, it'll complain extra parameter provided when calling get-credential with previous api version.